examples/api/lib/cupertino/date_picker/date_picker.0.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for CupertinoDatePicker
+
+import 'package:flutter/cupertino.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      title: _title,
+      home: MyStatelessWidget(),
+    );
+  }
+}
+
+class MyStatelessWidget extends StatefulWidget {
+  const MyStatelessWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatelessWidget> createState() => _MyStatelessWidgetState();
+}
+
+class _MyStatelessWidgetState extends State<MyStatelessWidget> {
+  DateTime date = DateTime(2016, 10, 26);
+  DateTime time = DateTime(2016, 5, 10, 22, 35);
+  DateTime dateTime = DateTime(2016, 8, 3, 17, 45);
+
+  @override
+  Widget build(BuildContext context) {
+    void _buildDialog(Widget child) {
```

Comment:
I would make this a member of the State object, and not call it `_buildDialog`, but `_showDialog` (or something indicating that it shows a dialog).